extensions/commands/recipe/README.md

```diff
@@ -36,3 +36,32 @@ updating moltenvk/1.2.0 to moltenvk/1.2.2 in conan-center-index/recipes/qt/6.x.x
 updating libjpeg-turbo/2.1.4 to libjpeg-turbo/2.1.5 in conan-center-index/recipes/qt/6.x.x/conanfile.py:386
 Successfully bumped the dependencies of recipe conan-center-index/recipes/qt/6.x.x/conanfile.py
 ```
+
+#### [Create SBOM](cmd_create_sbom.py)
+
+Creates an SBOM in CycloneDX 1.4 JSON format.
+
+**Parameters**
+* **format** _Required_: `cyclonedx_1.4_json`
+* supports all other arguments used by `conan graph`, see `conan recipe:create-sbom --help`
+
+**Dependencies**
+The command requires the packages `packageurl` and `cyclonedx`.
```

Comment:
do not depend on `cyclonedx`, this is not a library but a set of executables.
depend on `cyclonedx-python-lib` instead.